lib/js-api/src/api/content-rest-api/docs/JobIdBody.md

```diff
@@ -0,0 +1,7 @@
+# JobIdBody
+
+## Properties
+
+ Name      | Type   | Description                                                                           | Notes             
```

Comment:
Getting this issue for that table: 
<img width="1198" alt="image" src="https://github.com/user-attachments/assets/c4920c39-1797-4361-83d6-a041649b40f4">

Reply:
Since this file is removed, and its data moved to NodesApi.md file, this warning can be ignored